west.yml

```diff
@@ -125,6 +133,12 @@ manifest:
       path: modules/alexa-embedded
       revision: face92d8c62184832793f518bb1f19379538c5c1
       remote: alexa
+    - name: cddl-gen
+      remote: oyro
+      repo-path: cddl_gen
+      revision: master
```

Comment:
It isn't safe to track master, if it goes forward the older revision of NCS might not work.

Reply:
This will be resolved, please ignore west.yml for now. I updated the PR description to reflect this.